Assets/BossRoom/Scripts/Client/UI/PopupPanel.cs

```diff
@@ -59,6 +59,8 @@ public class PopupPanel : MonoBehaviour
 
         private const string k_DefaultConfirmText = "OK";
 
+        static readonly char[] k_InputFieldIncludeChars = new[] { '.' };

```

Comment:
Did you test this with relay? I think you're missing '_' no?

Reply:
Done.